itou/approvals/models.py

```diff
@@ -1180,6 +1180,8 @@ class ProlongationRequest(CommonProlongation):
         blank=True,
     )
 
+    reminder_sent_at = models.DateTimeField(verbose_name="rappel envoyé le", null=True, blank=True)
```

Comment:
Je doute qu’on ait un jour besoin de modifier ce champ.
```suggestion
    reminder_sent_at = models.DateTimeField(verbose_name="rappel envoyé le", null=True, editable=False)
```